main.py

```diff
@@ -1,10 +1,41 @@
 import init_django_orm  # noqa: F401
 
+import json
+
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players = json.load(file)
+    for name, info in players.items():
+        if not Race.objects.filter(name=info["race"]["name"]).exists():
+            Race.objects.create(
+                name=info["race"]["name"],
+                description=info["race"]["description"])
+        player_race = Race.objects.get(name=info["race"]["name"])
```

Comment:
This will not affect the number of database queries but may improve readability. For this, use the `get_or_create()` method. Just try it. The same in other similar cases

Reply:
Thanks. It's better now